libvast/vast/concept/printable/std/chrono.hpp

```diff
@@ -48,22 +46,47 @@ struct duration_printer : printer<duration_printer<Rep, Period, Policy>> {
 
   template <class R, class P>
   static std::string units(std::chrono::duration<R, P>) {
-    return date::detail::get_units<char>(Period{}).data();
+    if constexpr (Period::type::den == 1)
+      return "not implemented";
+    else
+      return "not implemented";
   }
 
-  template <class R>
-  static std::string units(std::chrono::duration<R, std::micro>) {
-    return "us";
+#define UNIT_SUFFIX(type, suffix)                                              \
+  template <class R>                                                           \
+  static std::string units(std::chrono::duration<R, type>) {                   \
```

Comment:
Does this function need to be `static`?

Reply:
It does not *need* to be static, but why should it not be? It does not need a value of `duration_printer` only the full type.

Reply:
Ah, I didn't see it's in a `struct`, sorry...